app/controllers/concerns/latest_version.rb

```diff
@@ -5,5 +5,9 @@ module LatestVersion
     def latest_version
       @latest_version ||= @rubygem.versions.most_recent
     end
+
+    def latest_version_by_slug
+      @latest_version = Version.find_by!(full_name: "#{params[:rubygem_id]}-#{params[:version_id]}")
```

Comment:
Can you please also look into including platform here? [nokogiri java](https://rubygems.org/gems/nokogiri/versions/1.10.3-java) has concourse and simplecov as dependencies but [nokogiri ruby](https://rubygems.org/gems/nokogiri/versions/1.10.3) doesn't.